src/mono/wasi/Makefile

```diff
@@ -0,0 +1,29 @@
+include Makefile.variable
```

Comment:
Shouldn't these files be under src/mono/wasm/wasi ?

Reply:
`src/mono/wasm` is full of things specific to Emscripten compilation and browser-based environments, none of which are used or relevant to the WASI case. I put it separate to clarify that there isn't a dependency from the WASI support on `src/mono/wasi`, at least as it stands currently. That could change in the future if some aspects of commonality like AOT compilation were shared.

If you think I should, I can move `src/mono/wasi` inside `src/mono/wasm`, but it would still really be an independent piece.

Reply:
>  and have `wasi` and `browser` as separate pieces within that. 

:+1: